gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -940,10 +939,10 @@ function GM:PostCleanupMap()
 end
 
 local function CleanUp()
-    game.CleanUpMap()
+    game.CleanUpMap(false, nil, function() ents.TTT.FixParentedPostCleanup() end)
```

Comment:
afaik such anonymous functions are pretty slow. You shoul define this as a local function that is only referenced here

Reply:
FWIW i don't know why rubat didn't just pass the invoked function directly

Reply:
![image](https://github.com/TTT-2/TTT2/assets/13639408/327e3ba7-bd34-4191-893a-4393e67cb56f)

I guess this might reduce the round end / round restart lag a bit. Not completely sure though. I'd prefer it if it were moved to a local function though and only referenced here

Reply:
idk why rubat did it this way, he could have just passed ents.TTT.FixParentedPostCleanup directly

Reply:
I think they did it because this way it is moved to the end of the tick. This is why I asked you to put this in a local function that is refernced